inst/table.m

```diff
@@ -500,9 +500,21 @@ function prettyprint (this)
       out = this;
     endfunction
     
-    function out = size_equal
+    ## -*- texinfo -*-
+    ## @node table.size_equal
+    ## @deftypefn {Method} {@var{out} =} size_equal (@var{varargin})
+    ##
+    ## True if the dimensions of all arguments agree.
+    ##
+    ## @end deftypefn
+    function out = size_equal (varargin)
       %SIZE_EQUAL True if the dimensions of all arguments agree.
-      error ('table.size_equal: size_equal is not yet implemented for tables');
+      [nr, nv] = size(varargin{1});
```

Comment:
I prefer a bit longer variable names in the Tablicious code. Could these be `nrows` and `nvars` instead?

Reply:
Done.